cmake-modules/AzureVcpkg.cmake

```diff
@@ -18,13 +18,50 @@ macro(az_vcpkg_integrate)
   endif()
 endmacro()
 
-macro(az_vcpkg_export targetName macroNamePart dllImportExportHeaderPath)
+macro(az_vcpkg_portfile_prep targetName fileName contentToRemove)
```

Comment:
Will this step happen **before** the sha512 is calculated?

Reply:
Yes, it is before.
So, our build process produces `out/<buildCfg>/vcpkg/ports/<libname>`. You are free to grab directories from there and put them to your local copy of vcpkg repo to `%VCPKG_ROOT%/ports/`, play with it locally, or even create a PR to VcPkg manually, but you need to update SHA first.
Daniel's automation grabs these files, replaces "SHA 1" with the actual SHA of the release, renerates version SHAs, and pushes the PR to VcPkg repo.
And this is why I think it is the right division of labor between us and EngSys, when it is us who drops the copyrights, because both files live in our repo, and should we update copyring, we would update the script that updates them too.
But everythign related to build release process itself is CentralEngSys, and it even lives in their repo.

Reply:
Otherwise, we would have to search for `"# Copyright (c) Microsoft Corporation. All rights reserved.\n# SPDX-License-Identifier: MIT\n\n"` (as today) on Linu/Mac, and for `"# Copyright (c) Microsoft Corporation. All rights reserved.\r\n# SPDX-License-Identifier: MIT\r\n\r\n"` on Windows.